pkg/plugin/provision/spirehelm/spirehelm.go

```diff
@@ -96,9 +97,11 @@ func (h *SpireHelm) deploy(ctx context.Context, ds datasource.DataSource, opts *
 		return err
 	}
 
-	if err := h.AddSPIRERepository(ctx, opts.KubeCfgFile, statusCh); err != nil {
-		return err
-	}
+	if repo, ok := os.LookupEnv("HELM_REPO_PATH"); ok && repo != "" {
+		statusCh <- provision.StatusOk("Deploying", fmt.Sprintf("Found HELM_REPO_PATH value, using local chart: %s", repo))
+	} else if err := h.AddSPIRERepository(ctx, opts.KubeCfgFile, statusCh); err != nil {
+			return err
+  }
```

Comment:
Whitespace seems off here

Reply:
Fixed, this seems to happen more often when accepting suggestions